src/Maker/MakeResetPassword.php

```diff
@@ -236,6 +238,18 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
             EntityManagerInterface::class,
         ];
 
+        // Namespace for ResetPasswordExceptionInterface was imported above
+        $problemValidateMessageOrConstant = \defined('SymfonyCasts\Bundle\ResetPassword\Exception\ResetPasswordExceptionInterface::MESSAGE_PROBLEM_VALIDATE')
+            ? 'ResetPasswordExceptionInterface::MESSAGE_PROBLEM_VALIDATE'
+            : "'There was a problem validating your password reset request'";
+        $problemHandleMessageOrConstant = \defined('SymfonyCasts\Bundle\ResetPassword\Exception\ResetPasswordExceptionInterface::MESSAGE_PROBLEM_VALIDATE')
+            ? 'ResetPasswordExceptionInterface::MESSAGE_PROBLEM_HANDLE'
+            : "'There was a problem handling your password reset request'";
+
+        if ($isTranslatorAvailable = interface_exists(TranslatorInterface::class)) {
```

Comment:
I think we should check for https://github.com/symfony/symfony/blob/6.1/src/Symfony/Component/Translation/Translator.php instead (but then we would inject the `TranslatorInterface` for the `use` statement as you have).

The reason is that `TranslatorInterface` is from `symfony/translations-contract`. So currently, this isn't checking to see if you have `symfony/translator` installed, but instead it's checking to see if you have `symfony/translations-contract` installed, which will ALWAYS be the case because that is a dependency of `symfony/validator`.

Reply:
Makes sense, changed